src/OrbitGl/ScopedStatus.h

```diff
@@ -83,11 +82,10 @@ class ScopedStatus final {
     if (std::this_thread::get_id() == data_->main_thread_id) {
       data_->status_listener->ClearStatus(data_->status_id);
     } else {
-      auto executor = data_->main_thread_executor.lock();
-      if (executor == nullptr) return;
-      executor->Schedule([status_listener = data_->status_listener, status_id = data_->status_id] {
-        status_listener->ClearStatus(status_id);
-      });
+      TrySchedule(data_->main_thread_executor,
+                  [status_listener = data_->status_listener, status_id = data_->status_id] {
+                    status_listener->ClearStatus(status_id);
+                  });
```

Comment:
this one looks different, you had a return here before. I guess this is correct now because we still want to reset data_

Reply:
Yes, you're right. My original solution was wrong.

Reply:
Glad that we talked about it. :sweat_smile: